ifrs17-template/Test/IfrsVariablesTest.ipynb

```diff
@@ -104,10 +113,32 @@
             "execution_count": 0,
             "outputs": []
         },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "## BenchMark to test Projections"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "var bmFilesTestProjection = new BenchmarkMetadata[]{",
```

Comment:
Move this BM in the Projection section

Reply:
Done. But wait for your answer in Teams regarding this suggestion.